proj/hps_accel/gateware/stream/actor.py

```diff
@@ -93,7 +93,11 @@ def __init__(self, input_type, output_type):
         super().__init__(input_type, output_type)
 
     def control(self, m: Module):
-        m.d.comb += flowcontrol_passthrough(self.sink, self.source)
+        m.d.comb += [
```

Comment:
I think this is a nice improvement (BinaryCombinatorialActor type instead of flowcontrol_passthrough function).